### PR TITLE
extend ERFA compatibility range

### PR DESCRIPTION
**Pull request by KlausC**
The last CI run resulted in the following error, which has been resolved by modifying the `[compat]` entry for a package.

```ERROR: Unsatisfiable requirements detected for package ERFA [17511681]:
 ERFA [17511681] log:
 ├─possible versions are: [0.4.0, 0.5.0, 0.6.0-0.6.3] or uninstalled
 ├─restricted to versions 0.4.0-0.4 by an explicit requirement, leaving only versions 0.4.0
 └─restricted by compatibility requirements with Compat [34da2185] to versions: [0.5.0, 0.6.0-0.6.3] or uninstalled — no versions left
   └─Compat [34da2185] log:
     ├─possible versions are: [1.0.0-1.0.1, 1.1.0, 1.2.0, 1.3.0, 1.4.0, 1.5.0-1.5.1, 2.0.0, 2.1.0, 2.2.0-2.2.1, 3.0.0, 3.1.0, 3.2.0, 3.3.0-3.3.1, 3.4.0, 3.5.0, 3.6.0, 3.7.0, 3.8.0, 3.9.0-3.9.1, 3.10.0, 3.11.0, 3.12.0, 3.13.0, 3.14.0, 3.15.0, 3.16.0, 3.17.0, 3.18.0, 3.19.0, 3.20.0, 3.21.0, 3.22.0, 3.23.0] or uninstalled
     └─restricted to versions 3.23.0 by an explicit requirement, leaving only versions 3.23.0
```

### CODE REVIEW

File: Project.toml [hunk 1]
```diff
@@ -12,7 +12,7 @@ MuladdMacro = "46d2c3a1-f734-5fdb-9937-b9b9aeba4221"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 
 [compat]
-ERFA = "^0.4"
+ERFA = "^0"
```
Comment (giordano) [inline Project.toml:15]: This is not acceptable, this includes an infinite number of breaking changes, you must list all of them explicitely

File: .gitignore [hunk 2]
```diff
@@ -4,3 +4,4 @@
 data/
 docs/build/
 Manifest.toml
+.vscode
```
Comment (giordano) [inline .gitignore:7]: Please add newlines at the end of files

**Review by giordano — CHANGES_REQUESTED** [review]

File: Project.toml [hunk 3]
```diff
@@ -12,7 +12,7 @@ MuladdMacro = "46d2c3a1-f734-5fdb-9937-b9b9aeba4221"
 Reexport = "189a3867-3050-52da-a836-e630ba90ab69"
 
 [compat]
-ERFA = "^0.4"
+ERFA = "^0.6"
```
Comment (giordano) [inline Project.toml:15]: Why not 0.4 and 0.5?  Also, the caret is redundant, it's the default

Comment (KlausC) [reply Project.toml:15]: because another dependency required at least 0.6
the caret is copying the existent style

**Comment by KlausC** [comment]
I think I made all requested changes - at least CI is content now...

**Comment by mileslucas** [comment]
looks good, thanks for the help!

**Pull request merged**